app/views/shared/export/_plan.erb

```diff
@@ -43,7 +43,7 @@
             <% blank = answer.present? ? answer.text.gsub(/<\/?p>/, '').gsub(/<br\s?\/?>/, '').chomp.blank? : true %>
             <% if blank && @show_unanswered %>
               <p><%= _('Question not answered.') -%></p>
-            <% else %>
+            <% elsif !blank %>
```

Comment:
rails helper method .present? would be a clearer solution instead of the baked variable blank we have defined above...

Reply:
it would be but I'm seeing a lot of `<p></p>` stored in the DB. I'm thinking that this is happening when the user blanks out the answer. That value always equates to present? true

Reply:
yep! If that's the case we should look at tinymce to clear out everything so the answer text becomes '' and therefore .present? is false